INTMDB-417-Network Container code change (#130)
mongodb/mongodbatlas-cloudformation-resources: cfn-resources/network-container/docs/README.md

_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)

#### AtlasCidrBlock

> CIDR block that Atlas uses for your clusters.

Reviewer: ```suggestion
IP addresses expressed in Classless Inter-Domain Routing (CIDR) notation that MongoDB Cloud uses for the network peering containers in your project. MongoDB Cloud assigns all of the project's clusters deployed to this cloud provider an IP address from this range. MongoDB Cloud locks this value if an M10 or greater cluster or a network peering connection exists in this project.

These CIDR blocks must fall within the ranges reserved per RFC 1918. AWS further limits the block to between the /24 and /21 ranges.

To modify the CIDR block, the target project cannot have:
- Any M10 or greater clusters
- Any other VPC peering connections

You can also create a new project and create a network peering connection to set the desired MongoDB Cloud network peering container CIDR block for that project. MongoDB Cloud limits the number of MongoDB nodes per network peering connection based on the CIDR block and the region selected for the project.

Example: A project in an Amazon Web Services (AWS) region supporting three availability zones and an MongoDB CIDR network peering container block of limit of /24 equals 27 three-node replica sets.
```

Unique identifier of the Network Peering container.

#### VpcId

> Unique identifier of the peer VPC.

Reviewer: ```suggestion
Unique string that identifies the MongoDB Cloud VPC on AWS.
```

#### Provisioned

> Flag that indicates if the project has clusters deployed in the Network Peering container.

Reviewer: ```suggestion
Boolean flag that indicates whether MongoDB Cloud clusters exist in the specified network peering container.
```
